run-script-os

```diff
@@ -0,0 +1 @@
+[B[B
```

Comment:
Is this file intentional? @aaronlippold

Reply:
yes the thats managed by another plugin, it works, since your `npm run test` works.

Reply:
Found it, Fixed it.